utils/tf_solver.py

```diff
@@ -39,7 +39,7 @@ def skip(self, model_init_fn, dataset, normalization):
             return True, 'Cannot use both decoupled and coupled weight decay'
         return False, None
 
-    def set_objective(self, model_init_fn, dataset, normalization):
+    def get_lr_wd_cback(self, n_epochs=200):
```

Comment:
I would call this `max_epochs`, as we are not sur to reach it.

Reply:
sure